src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,17 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * <p>This method sends a message as if it came from this player.
+     * It does not send a message to this player.</p>
+     *
+     * @param message The message to send
+     * @return The event that was thrown from sending the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
Your changes fixed most of my concerns. However it still does not explicitly states whether this method can be used to send commands.
Maybe there should be a hint which method can be used to send commands because both features are somehow related.